### DISCUSSION TITLE
unfilled checkmarks and swiping through days

### DISCUSSION THREAD

**Discussion by myghusernamegoeshere** [iSoron/uhabits#1544] [Feature Requests]
Hi there
First of all thanks for this great app, I've tried many other apps in this category and this one is the best.

- I've got ADHD and the "unfilled checkmarks for the non-daily habbits" makes it REALLY hard for me to see which habbits I've done. Would you kindly make this feature optional?

- And would you make it available for the user to swip beteen weeks? it would be really helpful if we could swip and see how we have performed on the weeks gone.

Thank you in advance

**Comment by iSoron** [comment]
> I've got ADHD and the "unfilled checkmarks for the non-daily habbits" makes it REALLY hard for me to see which habbits I've done. Would you kindly make this feature optional?

You can disable automatic checkmarks by making your habit daily.

> And would you make it available for the user to swip beteen weeks? it would be really helpful if we could swip and see how we have performed on the weeks gone.

On the main screen, you can already swipe the header (the grey row at the top with the dates) to see previous days.
